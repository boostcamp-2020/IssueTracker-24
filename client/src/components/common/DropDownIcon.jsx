import React, { useState } from 'react';
import styled from 'styled-components';

const DropDownWrapper = styled.div`
  display: inline-flex;
  align-items: center;
  justify-content: center;
  vertical-align: middle;
`;

const DropDownIcon = () => {
  return (
    <DropDownWrapper className="material-icons">
      arrow_drop_down
    </DropDownWrapper>
  );
};

export default DropDownIcon;
