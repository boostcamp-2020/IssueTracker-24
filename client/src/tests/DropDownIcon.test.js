import React from 'react';
import { render } from '@testing-library/react';
import DropDownIcon from '../components/common/DropDownIcon';

describe('<DropDownIcon/>', () =>{
    it('matches snapshot', () =>{
        const utils = render(<DropDownIcon/>);
        expect(utils.container).toMatchSnapshot();
    });
})