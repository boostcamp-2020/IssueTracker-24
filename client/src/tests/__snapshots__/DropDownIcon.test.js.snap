// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropDownIcon/> matches snapshot 1`] = `
<div>
  <div
    class="sc-bdnylx gPQxjM material-icons"
  >
    arrow_drop_down
  </div>
</div>
`;
